src/test/java/com/google/api/codegen/protoannotations/testdata/java_library.baseline

```diff
@@ -76,7 +76,7 @@ task allJars(type: Copy) {
   // Replace with `from configurations.testRuntime, jar` to include test dependencies
   from configurations.runtime, jar
 }
-============== file: src/main/java/com/google/gcloud/pubsub/v1/LibraryClient.java ==============
+============== file: src/main/java/com/google/example/library/v1/LibraryClient.java ==============
```

Comment:
Why is `pubsub` not part of the baseline? I know we had talked about making the baselines smaller, but I assumed that would be done separately from the annotations. Could you bring me up to speed?

Reply:
This is just copying over the java_library.baseline from the gapic test source folder. That change was made inhttps://github.com/googleapis/gapic-generator/pull/2399

Reply:
This java_library.baseline isn't yet used, but it will be soon.

Reply:
To recap our discussion: this is a separate baseline that (for now) you're keeping manually in sync with the original baseline hzyi updated in #2399---you're just copying those changes into here.